tests/org.eclipse.swt.tests.gtk/ManualTests/org/eclipse/swt/tests/gtk/snippets/Issue644_CompleteTableEditor.java

```diff
@@ -100,51 +109,43 @@ public static void main(String[] args) {
 		display.dispose();
 	}
 
-	protected static void populateColumnName(int columnNum, TableColumn column) {
```

Comment:
Don't really see a point in changing the switch? But probably you want to add the new method after the big switch methods, so that the history isn't as mangled as it is right now.

Reply:
The code is replaced with **one line**. Other switches were using indexes and I hate updating indexes every time because they do not have any meaning.